CHANGELOG.md

```diff
@@ -1,11 +1,42 @@
-## Next release
-
-This release includes changes to amtool which are not fully backwards compatible with the previous
-amtool version (#1798) related to backup and import of silences.  If a backup of silences is created using 
-a previous version of amtool (v0.16.1 or earlier), it is possible that not all silences can be correctly imported using a later version of amtool.
-
-* [CHANGE] Make api/v2/status.cluster.{name,peers} properties optional for Alertmanager with disabled clustering (#1728)
+## 0.17.0 / 2019-05-02
+
+This release includes changes to amtool which are not fully backwards
+compatible with the previous amtool version (#1798) related to backup and
+import of silences. If a backup of silences is created using a previous
+version of amtool (v0.16.1 or earlier), it is possible that not all silences
+can be correctly imported using a later version of amtool.
+
+Additionally, the groups endpoint that was dropped from api v1 has been added
+to api v2. The default for viewing alerts in the UI now consumes from this
+endpoint and displays alerts grouped according to the groups defined in the
+running configuration. Custom grouping is still supported.
+
+* [FEATURE] Add groups endpoint to v2 api (#1791)
+* [ENHANCEMENT] Set HTTP headers to prevent asset caching (#1817)
+* [ENHANCEMENT] Make api/v2/status.cluster.{name,peers} properties optional for Alertmanager with disabled clustering (#1728)
```

Comment:
This is a _change_, as model generations might change with the properties being optional.

Reply:
E.g. the server side changes broke our ui implementation.